src/NHSD.BuyingCatalogue.Ordering.Api/Models/ServiceRecipientModel.cs

```diff
@@ -10,12 +10,6 @@ public ServiceRecipientModel()
         {
         }
```

Comment:
We can remove the empty constructor and `using NHSD.BuyingCatalogue.Ordering.Domain;` now.